http/transport.go

```diff
@@ -502,11 +518,14 @@ func handleIsConnected(s api.FluxService) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		inst := getInstanceID(r)
 
-		switch s.IsDaemonConnected(inst) {
-		case platform.ErrPlatformNotAvailable:
-			w.WriteHeader(http.StatusNotFound)
-		case nil:
+		err := s.IsDaemonConnected(inst)
+		if err == nil {
 			w.WriteHeader(http.StatusNoContent)
+			return
+		}
+		switch err.(type) {
+		case flux.UserConfigProblem:
+			w.WriteHeader(http.StatusNotFound)
```

Comment:
Not quite; `errorResponse` would write a 422 Unprocessable Entity (as a stand in for "your request is fine, I just can't do it right now"), whereas this is expected to return No Content, Not Found, or Aaaaargh.